### DISCUSSION TITLE
【读书会】《巴菲特之道》 - 第1期 | ApacheCN

### DISCUSSION THREAD

**Discussion by yanqiang921106** [apachecn/home#235] [Ideas]
# <center>【读书会】《巴菲特之道》 - 第1期 | ApacheCN<center>

* 【推荐】读书会: 《巴菲特之道》译者：杨天南
* 【推荐】留言地址: https://github.com/apachecn/home/issues/199

> 参与

* QQ群【临时群】: 1062944139
* 负责人紫色木槿: 1076003661 (QQ)

> 时间

**每两周三晚8点，月初会发布本月阅读书籍2-4本，顺序阅读。

* 时间周期: 2021-07-07 ～ 2021-07-21 （共 14 天，对自己严格的建议控制在7天内）
* 语音频率: 每两周三晚9点语音一次，周末休息跳过
* **群友发言：**5~10人，5分钟/人，计划时长0.5-1小时，带着问题发言	
* **[@紫色木槿](https://github.com/yanqiang921106)** 会进行语音: 沟通进度和督促学习

> 内容

1. 分享读书心得
2. 学到的知识
3. 可以应用到实际工作生活中的
4. 自己的应用计划			

每一本书，专门一个issues，供大家留言分享		

> 活动目的

1. 培养自己对理财的正确认识
2. 选出2个人作为大肥羊学校的学员，作为其他活动的代理班长		

---

* 总结：一人可以走的很快，但是一群人却可以走的很远

**Comment by jiangzhonglian** [comment]
1
